client/blocks/reader-post-card/index.jsx

```diff
@@ -86,27 +86,27 @@ export default class RefreshPostCard extends React.Component {
 		const { post, originalPost, site, feed, onCommentClick, showPrimaryFollowButton } = this.props;
 		const isPhotoOnly = !! ( post.display_type & DisplayTypes.PHOTO_ONLY );
 		const isGallery = !! ( post.display_type & DisplayTypes.GALLERY );
-		const featuredAsset = ( <FeaturedAsset post={ post } /> );
+		const featuredAsset = FeaturedAsset( { post } ); // this is ... gross? gross.
```

Comment:
it's using a "component" as something other than a component. If we want to walk this way, it should be `buildFeaturedAsset` instead of `FeaturedAsset`.